recipe/meta.yaml

```diff
@@ -122,7 +145,7 @@ outputs:
       license_file: LICENSE
 
   - name: gxx_{{ cross_target_platform }}
-    script: install-g++.sh
+    script: install-gxx.sh
```

Comment:
Let's not do random refactorings in PR.

Reply:
A trivial file rename (for consistency with the outputs resp. the other big compiler activation feedstock) can hardly be called a refactor... You could check the individual commits, which separate the various concerns. But if you insist, sure.